nexus/reconfigurator/planning/src/blueprint_builder/builder.rs

```diff
@@ -2140,12 +2231,12 @@ pub mod test {
         // The example blueprint should have internal NTP zones on all the
         // existing sleds, plus Crucible zones on all pools.  So if we ensure
         // all these zones exist, we should see no change.
-        for (sled_id, sled_resources) in
-            example.input.all_sled_resources(SledFilter::Commissioned)
+        for (sled_id, sled_details) in
+            example.input.all_sleds(SledFilter::Commissioned)
```

Comment:
Why did we switch from `all_sled_resources` here? It doesn't look like we use any of the non-resource details. (Same comment on some of the other similar changes below)

Reply:
Good eye. I *think* this was leftover from an old commit before I had started reworking diffs. Reverted in de4a0805309651eea73e6c8bab1e256afbaf992d.